File: problem-set-1/mario/more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    //Do-while loop for user's input.
    int h;
    do
    {
        h = get_int("Height: ");
    }
    while (h < 1 || h > 8);

    //While loop with four for loops nested within for printing two pyramids with a gap in between of them.
    int x = 1;
    while (x <= h)
    {
        for (int a = h - x; a > 0; a--)
        {
            printf(" ");
        }
        for (int b = 1; b <= x; b++)
        {
            printf("#");
        }
        for (int c = 0; c < 2; c++)
        {
            printf(" ");
        }
        for (int d = 1; d <= x; d++)
        {
            printf("#");
        }
        printf("\n");
        x++;
    }
}

File: problem-set-6/mario/more/mario.py
import cs50

# Improvised do-while loop for users input
h = cs50.get_int("Height: ")

while h < 1 or h > 8:
    h = cs50.get_int("Height: ")


# While loop for printing two opposing pyramids with a gap in between of them
h2 = h

while h2 != 0:
    i = h2 - 1
    print(" " * i + "#" * (h - i) + "  " + "#" * (h - i))
    h2 -= 1
